crates/compiler/build/src/link.rs

```diff
@@ -103,21 +103,6 @@ pub fn get_relative_path(sub_path: &Path) -> Option<PathBuf> {
     None
 }
 
-fn find_zig_glue_path() -> PathBuf {
```

Comment:
Zig 0.13.0 removed/modified the `--mod` and `--deps` and so we can no longer just include these as a module. It may be possible, but we didn't find a way to make this work. The solution we have gone with is to copy the zig builtins source into each test platform once per test run, which works reliably.